nbclient/output_widget.py

```diff
@@ -0,0 +1,85 @@
+from ipykernel.jsonutil import json_clean
```

Comment:
Indeed, this seems like a weak reason to depend on ipykernel.

Should we duplicate the implementation of `json_clean` in this package instead?

Reply:
There is a jsonutils in `jupyter_client` as well but without that function.

How about adding a jsonutils here?